index.html

```diff
@@ -0,0 +1,139 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width" />
+  <title>Jacob Schatz Website</title>
+  <link href="https://fonts.googleapis.com/css?family=Open+Sans&display=swap" rel="stylesheet" />
+  <link href="assets/css/styles.css" rel="stylesheet" type="text/css" />
+
+</head>
+
+<body>
+  <header>
+    <div id="hero-image-container">
+      <img id="header-logo" src="images/handsome_man.jpg" alt="hero image" />
+      <p class="name-container">Jacob Schatz</p>
+   <!--   
+      <ul>
+
+        <li class="hero-first-name"><a> J </a></li>
+        <li class="hero-first-name"><a>A </a></li>
+        <li class="hero-first-name"><a>C </a></li>
+        <li class="hero-first-name"><a>O </a></li>
+        <li class="hero-first-name"><a>B</a></li>
+      </ul>
+      <ul>
+
+        <li class="hero-last-name"><a> S </a></li>
+        <li class="hero-last-name"><a>C </a></li>
+        <li class="hero-last-name"><a>H </a></li>
+        <li class="hero-last-name"><a>A </a></li>
+        <li class="hero-last-name"><a>T</a></li>
+        <li class="hero-last-name"><a>Z</a></li>
+      </ul>
+      -->
+    </div>
+    <div id="group">
+      <nav id="nav-bar">
+        <ul>
+          <li class="nav-buttons" id="homeNav"><a>Home </a></li>
+          <li class="nav-buttons" id="homeNav"><a>| </a></li>
+          <li class="nav-buttons" id="aboutNav"><a>About </a></li>
+          <li class="nav-buttons" id="homeNav"><a>| </a></li>
+          <li class="nav-buttons" id="contactNav"><a>Contact</a></li>
+        </ul>
+      </nav>
+
+      <div id="social-media">
+        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" width="30" height="30">

```

Comment:
You can load a svg file through an `img` tag. https://css-tricks.com/using-svg/#article-header-id-2

That whole article is GOLD.